src/jdk.compiler/share/classes/com/sun/source/util/DocTreeFactory.java

```diff
@@ -336,6 +337,16 @@ default ReturnTree newReturnTree(boolean isInline, List<? extends DocTree> descr
      */
     SnippetTree newSnippetTree(List<? extends DocTree> attributes, TextTree text);
 
+    /**
```

Comment:
I see there is a missing `@since` tag, which I'll fix.
I'd prefer to leave the extra whitespace for now, and maybe cleanup the style of the other comments later.

Reply:
Added @since. Removed the whitespace for now.